src/global-styles.ts

```diff
@@ -73,10 +74,8 @@ export const globalStyles = css`
 
     // Action
     --action: var(--earth400);
-    --actionBackground: var(--earth50);
+    --actionBackground: ${transparentize(color.action, opacity.statusBackgroundColor)};
```

Comment:
Should this always have an opacity? It introduces this bug again:

<img width="1035" alt="CleanShot 2023-05-23 at 15 27 21@2x" src="https://github.com/TicketSwap/solar/assets/14276144/482f5ecb-28e3-4713-9fb8-89ba07892a1e">

Reply:
Yeah, will use a new `activeBackground` here instead so we can distinguish the two use-cases